x-pack/plugin/enrich/src/main/java/org/elasticsearch/xpack/enrich/EnrichStore.java

```diff
@@ -153,18 +155,14 @@ public static EnrichPolicy getPolicy(String name, ClusterState state) {
      * Gets all policies in the cluster.
      *
      * @param state the cluster state
-     * @return a Map of <code>policyName, EnrichPolicy</code> of the policies
+     * @return a read-only Map of <code>policyName, EnrichPolicy</code> of the policies
      */
     public static Map<String, EnrichPolicy> getPolicies(ClusterState state) {
-        final Map<String, EnrichPolicy> policies;
         final EnrichMetadata enrichMetadata = state.metadata().custom(EnrichMetadata.TYPE);
```

Comment:
It's a small thing, but look at the pattern for `IndexLifecycleMetadata` where there's a `static final` `EMPTY` constant. Then you can update this call to use the two-argument arity and skip the null check. And then you can look at the other uses of the one argument arity and fix them, too, I believe there are two of them.

Reply:
Eh, I just went for it in 0c798e02c7535ca06406c33eec9007ddd38ed152.